zellij-client/src/input_handler.rs

```diff
@@ -54,6 +62,10 @@ impl InputHandler {
         let alt_left_bracket = vec![27, 91];
         let bracketed_paste_start = vec![27, 91, 50, 48, 48, 126]; // \u{1b}[200~
         let bracketed_paste_end = vec![27, 91, 50, 48, 49, 126]; // \u{1b}[201
+
+        if !self.config.options.disable_mouse_mode {
```

Comment:
Since this is the direct `config` struct it doesn't merge the config options,
that would be the `options` struct you can see in `opts` in `main.rs`.
Maybe it would make sense to also merge the options back into the config
struct? Or maybe it makes more sense to pass the merged `options` struct.

Reply:
`input_handler` is also reading `keybinds` from `config`, so maybe passing both is best?